build.gradle

```diff
@@ -293,22 +293,57 @@ allprojects {
     }
   }
 
+  ext.withReleaseBuild = { Closure config ->
+    if(buildParams.snapshotBuild == false) {
+      config.call()
+    }
```

Comment:
For my education: I assume this makes it so that we end up depending on the `assemble` task only in release builds, correct? How does `config.call()` make this work?